openedx/features/course_experience/templates/course_experience/course-home-fragment.html

```diff
@@ -61,9 +61,9 @@ <h2 class="hd hd-3 page-title">${course.display_name_with_default}</h2>
                     ${HTML(course_home_message_fragment.body_html())}
                 % endif
 
-                % if welcome_message_fragment and UNIFIED_COURSE_TAB_FLAG.is_enabled(course.id):
+                % if message_fragment and UNIFIED_COURSE_TAB_FLAG.is_enabled(course.id):
                     <div class="section section-dates">
```

Comment:
@dianakhuang: I'd rename `section-dates` to`section-update-message` to match the fragment variable and satisfy @andy-armstrong's request.